src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -84,6 +84,18 @@ $dropdown-white: clr-getColor(lightest);
             visibility: hidden;
             z-index: map-get($clr-layers, dropdown-menu);
 
+            .dropdown-menu {
```

Comment:
As per discussion offline, I'm leaving this in as is. This helps support backward compatibility until the markup for dropdown < 0.8.13 is deprecated.